frontend/assets/results.js

```diff
@@ -26,12 +34,28 @@ function makeNewResult(dataFromFetch) {
       resultSection.querySelector('.completion-title').textContent = 'Completed: ';
       formattedTimestamp = processTimestamp(dataFromFetch[i].timestamp)
       resultSection.querySelector('.completion-date-time-data').textContent = formattedTimestamp
+      resultSection.addEventListener('click', e => {
```

Comment:
please remove this piece of code `resultSection.addEventListener('click', e => {
        expandShrink(e)
      })`